VideoLocker/src/test/java/org/edx/mobile/discussions/DiscussionTextUtilsTest.java

```diff
@@ -0,0 +1,130 @@
+package org.edx.mobile.discussions;
```

Comment:
Does JUnit4 run tests outside of the tests package? Also, we might want to have a discussion on where we would want to keep some tests i.e. in the feature folder or in a common folder.